darwin/backend_v2.py

```diff
@@ -148,15 +148,17 @@ def export_dataset(
         team_slug: Optional[str] = None,
     ):
         payload = {
-            "filters": filters,
             "format": format,
             "include_authorship": include_authorship,
             "include_export_token": include_token,
             "name": name,
-            "annotation_filters": {}
+            "annotation_filters": {},
         }
         if annotation_class_ids:
             payload["annotation_filters"] = {"annotation_class_ids": annotation_class_ids}
+        if filters is not None:
```

Comment:
should `annotation_filters` behave in the same way?

Reply:
Well, they do already. But even if not, that wouldn't be a problem, there are no magical, implicit `annotation_filters` like in case of regular `filters`.